x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/index/store/SearchableSnapshotDirectoryTests.java

```diff
@@ -380,10 +405,86 @@ private void testIndexInputs(final CheckedBiConsumer<IndexInput, IndexInput, Exc
         });
     }
 
+    public void testClearCache() throws Exception {
```

Comment:
This test comes from the now removed `CacheDirectoryTests` class with very few adjustments for the `SearchableSnapshotDirectory` instantiation.